integration-tests/http/__tests__/order/admin/rma-flows.spec.ts

```diff
@@ -0,0 +1,801 @@
+
+        orderResult = (await api.get(`/admin/orders/${order.id}`, adminHeaders))
+          .data.order
+
+        // Totals summary after all
+        expect(orderResult.summary).toEqual(
+          expect.objectContaining({
+            paid_total: 131,
+            difference_sum: -20,
+            refunded_total: 10,
+            transaction_total: 121,
+            // TODO: This looks wrong
+            pending_difference: -10,
+            current_order_total: 111,
+            original_order_total: 131,
+            // This is now positive as we have an item to be fulfilled
+            temporary_difference: 15,
+          })
+        )
+
+        // Lets create one more claim without fulfilling the previous one
+        claimWithInboundAndOutbound = (
+          await api.post(
+            "/admin/claims",
+            {
+              order_id: order.id,
+              type: ClaimType.REPLACE,
+              description: "Base claim",
+            },
+            adminHeaders
+          )
+        ).data.claim
+
+        orderResult = (await api.get(`/admin/orders/${order.id}`, adminHeaders))
+          .data.order
+
+        // Nothing changes from the previous expectation
+        expect(orderResult.summary).toEqual(
+          expect.objectContaining({
+            paid_total: 131,
+            difference_sum: -20,
+            refunded_total: 10,
+            transaction_total: 121,
+            pending_difference: -10,
+            current_order_total: 111,
+            original_order_total: 131,
+            temporary_difference: 15,
+          })
+        )
+
+        inboundItem = orderResult.items[0]
+
+        await api.post(
+          `/admin/claims/${claimWithInboundAndOutbound.id}/inbound/items`,
+          { items: [{ id: inboundItem.id, quantity: 1 }] },
+          adminHeaders
+        )
+
+        await api.post(
+          `/admin/claims/${claimWithInboundAndOutbound.id}/outbound/items`,
+          {
+            items: [
+              {
+                variant_id: product.variants[0].id,
+                quantity: 1,
+              },
+            ],
+          },
+          adminHeaders
+        )
+
+        await api.post(
+          `/admin/claims/${claimWithInboundAndOutbound.id}/request`,
+          {},
+          adminHeaders
+        )
+
+        orderResult = (await api.get(`/admin/orders/${order.id}`, adminHeaders))
+          .data.order
+
+        expect(orderResult.summary).toEqual(
+          expect.objectContaining({
+            paid_total: 131,
+            // TODO: This looks wrong
+            difference_sum: -10,
+            refunded_total: 10,
+            transaction_total: 121,
+            // TODO: This looks wrong
+            pending_difference: 15,
+            // TODO: This looks wrong
+            current_order_total: 136,
+            // TODO: This looks wrong
+            original_order_total: 146,
+            // TODO: This looks wrong
+            temporary_difference: 15,
```

Comment:
when I create 2 claims and none are fulfilled, the totals don't seem to be correct at all.